packages/calcite-components/src/components/color-picker/color-picker.e2e.ts

```diff
@@ -2236,9 +2243,64 @@ describe("calcite-color-picker", () => {
       const hueSliderScope = await page.find(`calcite-color-picker >>> .${CSS.hueScope}`);
 
       expect(await hueSliderScope.getComputedStyle()).toMatchObject({
-        top: "10px",
-        left: "0px",
+        top: "9.5px",
+        left: "-0.5px",
       });
     });
   });
+
+  describe("mouse interaction with scope", () => {
```

Comment:
Awesome coverage here. Can you move this closer to the keyboard scope interaction block? Bonus points if these are grouped within a `scope interaction` describe block.

```tsx
describe("scope interaction", () => {
  describe("mouse", async () => { /* ... */ });
  describe("keyboard", async () => { /* ... */ });
})
```

This is not critical and can be done in a follow-up PR.

Reply:
currently `mouse interaction with scope` is placed near to keyboard scope interaction block.

Reply:
will group them.